src/metaAnalysis.ts

```diff
@@ -1,30 +1,115 @@
 import { AirtableRecord, AggregationFactor, AggregatedRecord } from "./types"
+import { faPassport } from "@fortawesome/free-solid-svg-icons";
+
+// BROKEN NOW: CONFIDENCE INTERVAL TOOLTIPS 
 
 const Z_SCORE: number = 1.96;
 
-export function aggregateRecords(records: AirtableRecord[]) {
+function transformPrevalence (p: number, N: number, method: string){
+    switch(method as any) {
+        case "inverse_variance": {
+            return p; 
+        }
+        case "logit": {
+            return Math.log(p / (1 - p))
+        }
+        case "arcsin": {
+            return Math.asin(Math.sqrt(p))
+        }
+        case "double_arcsin": {
+            let n = N * p
```

Comment:
not changing the definition of n after its declaration, so we can just make it a const

Reply:
done